tests/DbTestCase.php

```diff
@@ -519,13 +519,15 @@ protected function createForm(FormBuilder $builder): Form
         }
 
         // Create destinations
-        foreach ($builder->getDestinations() as $itemtype => $destination_data) {
-            $destination = $this->createItem($itemtype, $destination_data);
-            $this->createItem(Form_FormDestination::class, [
-                'forms_forms_id' => $form->getID(),
-                'itemtype' => $destination->getType(),
-                'items_id' => $destination->getID(),
-            ]);
+        foreach ($builder->getDestinations() as $itemtype => $destinations) {
+            foreach ($destinations as $destination_data) {
+                $destination = $this->createItem($itemtype, $destination_data);
```

Comment:
Fixed bad implementation that prevented created multiple destinations of the same type.